packages/typespec-ts/test/azureModularIntegration/generated/versioning/typeChangedFrom/src/index.d.ts

```diff
@@ -13,11 +13,12 @@ export declare interface TestOptionalParams extends OperationOptions {
 export declare class TypeChangedFromClient {
     private _client;
     readonly pipeline: Pipeline;
-    constructor(endpointParam: string, version: Versions, options?: TypeChangedFromClientOptionalParams);
+    constructor(endpointParam: string, options?: TypeChangedFromClientOptionalParams);
     test(body: TestModel, param: string, options?: TestOptionalParams): Promise<TestModel>;
 }
 
 export declare interface TypeChangedFromClientOptionalParams extends ClientOptions {
+    version?: Versions;
```

Comment:
This change is introduced by tcgc fix and for more cases support for api version, i create an issue to track the effort: https://github.com/Azure/autorest.typescript/issues/3267.